src/alto-ui/Tag/Tag.scss

```diff
@@ -0,0 +1,48 @@
+@import '../scss/inc';
+
+.Tag {
+  background-color: $blue-10;
+  border: 1px solid $blue-20;
+  color: $blue;
+  border-radius: $border-radius-default;
+  padding: $spacing-x-small $spacing-small;
+  display: flex;
+  align-items: center;
+  line-height: 1;
+
+  &--button {
+    @extend %reset-button;
+    transition: all $transition;
+    cursor: pointer;
+
+    &:focus,
+    &:hover {
+      border-color: $blue;
+    }
+  }
+
+  &--rounded {
+    border-radius: 0.5rem + $spacing-x-small;
```

Comment:
` border-radius: 0.5rem + $spacing-x-small;` ?? Why ?
maybe sth from 
```
// ===================================
// Border Radius

$border-radius-default: 3px !default;
$border-radius-small: 0.125rem !default;
$border-radius-medium: 0.25rem !default;
$border-radius-large: 0.5rem !default;
$border-radius-circle: 50% !default;
$border-radius-pill: 15rem !default;
```

Reply:
`$border-radius-circle` cant work here because it's not a square. In those cases, you need divide the height by 2. But I can try to make this more understandable.

Reply:
do you thnk it is more clear:

```scss
    border-radius: ($font-size-medium + 2 * $spacing-x-small) / 2;
```
?

Reply:
one and another is not clear and in that case maye simple define some value.

Reply:
```scss
$padding-top: $spacing-x-small;
$padding-bottom: $spacing-x-small;
$padding-vertical: $padding-top + $padding-bottom;
$line-height: 1;
$height: $font-size-medium * $line-height + $padding-vertical;
border-radius: $height / 2;
```
?